ui/src/components/App.scss

```diff
@@ -0,0 +1,30 @@
+.platform-item > .platform-item-icon {
```

Comment:
I think that this is not correct location for the styles. I should move it somewhere else... Not sure where?

Reply:
At the moment if we need our own `sass` in a `js` file we add a `scss` file specifically for that `js`